.config/mypy/mypy.ini

```diff
@@ -8,6 +8,9 @@ ignore_missing_imports = True
 
 # Layers specific config
 
+[mypy-scapy.arch.*]
+implicit_reexport = True
```

Comment:
This is required because we have star imports in `scapy.arch` we can hardly get rid of (used for polyfill)